tests/cases/standalone/common/alter/alter_table.result

```diff
@@ -192,3 +192,46 @@ DROP TABLE phy;
 
 Affected Rows: 0
 
+CREATE TABLE grpc_latencies (
```

Comment:
This test likely doesn't cover the intended fixes; it's just a simple alter test. To properly cover this, we need a test that simulates both successful and failed alter region requests in distributed mode, that would make sense.

Reply:
The test covered in #5796. However, the PR also introduces some features(i.e., the consistency guard) from #5798. The proper tests you mentioned will be introduced in subsequent PRs.